packages/react-components/react-components/src/Concepts/Accessibility/FocusIndicator.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
+import { ScenariosListLink } from '../../AccessibilityScenarios/utils';
+
+<Meta title="Concepts/Developer/Accessibility/Focus indicator" />
+
+## Focus indicator
+
+Fluent UI components use [tabster](https://github.com/microsoft/tabster) for focus handling functionality, so that they can be easily integrated with application-level tabster functionality such as delooser and cross-iframe focusing.
+
+Fluent UI `@fluentui/react-tabster` package defines `useKeyboardNavAttribute`, `createCustomFocusIndicatorStyle` and `createFocusOutlineStyle` to integrate tabster keyboard navigation mechanism seemlesly within Fluent UI.
+
+### useKeyboardNavAttribute
+
+Instantiates [keyborg](https://github.com/microsoft/keyborg) and add attribute to ensure focus indicator synced to keyborg logic.
+
+`useKeyboardNavAttribute` is being used internally by Fluent UI providers and portals to ensure that top level elements will have attributes to indicate focus, in a similar way to what `:focus-visible` does, ensuring Fluent UI components are aware when focus is required.
+
+### createCustomFocusIndicatorStyle and createFocusOutlineStyle
```

Comment:
```suggestion
### Styling focus indicators
```

You have subheadings later for each of these functions, let's be more mission focused in the title